libparsec/crates/client/tests/unit/certificates_ops/add_revoked_certificate.rs

```diff
@@ -0,0 +1,183 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
better to name this file after the actual certificate name (so `add_user_revoked_certificate.rs`)

Reply:
Done